notebooks/binary_predicates.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        [display(widgets.HBox(row)) for row in rows]
</pre><p>There's not need to use list comprehension here. Just</p><pre class="ql-syntax" spellcheck="false">for row in rows:
	display(widgets.HBox(row))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>